package.json

```diff
@@ -20,6 +21,7 @@
     "immutable": "3.x.x"
   },
   "devDependencies": {
+    "@types/draft-js": "^0.7.33",
```

Comment:
It's used in `typings/index.d.ts` and `draft-js-export-html-tests.ts` as `import draftjs = require("draft-js");` -- in the first file, it just helps IDE with autocompletion and in the second one it is required during compilation, that's why it's a dev dependency.